src/include/settings/settings.h

```diff
@@ -22,12 +22,14 @@
 SETTING_int(port,
            "Peloton port (default: 15721)",
            15721,
+           512, 64000,
```

Comment:
See [](url)https://en.wikipedia.org/wiki/Registered_port
Lower bound on acceptable port value should be 1024
Upper bound should be 65535

Strictly speaking, we should not be allowing selection of the 1024 to 49151 range, except for registered or unassigned ports. 15721 is, AFAIK, not registered, but is an unassigned value. So, lets go with the values I listed above.